File: 프로그래머스/lv0/120835. 진료 순서 정하기/진료 순서 정하기.js
function solution(emergency) {
    let answer = [...emergency]; // emergeny배열을 복사하여 새로운 배열인 answer에 할당
    answer.sort((a, b) => b-a); // answer배열을 내림차순으로 정렬
    return emergency.map(n => answer.indexOf(n) + 1);   // 순위는 1부터 시작하므로 1을 더해줌
}

File: 프로그래머스/lv0/120835. 진료 순서 정하기/진료 순서 정하기.java
class Solution {
    public int[] solution(int[] emergency) {
        int[] answer = new int[emergency.length];       
        for (int i = 0; i < emergency.length; i++) {
            answer[i]++;    // answer[i]를 1로 초기화
            for (int j = 0; j < emergency.length; j++) {
                if (emergency[i] < emergency[j]) { // 응급도가 더 낮은 경우 응급도의 크기를 증가
                    answer[i]++;
                } 
            }
        }                    
        return answer;
    }
}